server/src/internalClusterTest/java/org/opensearch/search/knn/DenseVectorIT.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
This is new code, isn't it? Doesn't need an ES license grant.

Reply:
Yes, I'm adding several new classes. I actually wasn't sure on which one to use, checked few recent check-ins and classes in the same package, they used ES style license. 
Let me update PR and exclude ES part

Reply:
Only SPDX header is required for new code AFAIK.